terraform/implementation/main.tf

```diff
@@ -250,6 +259,54 @@ resource "helm_release" "agic" {
   ]
 }
 
+# Cert Manager
+
+resource "helm_release" "cert_manager" {
+  name             = "cert-manager"
+  repository       = "https://charts.jetstack.io"
+  chart            = "cert-manager"
+  namespace        = "cert-manager"
+  create_namespace = true
+  depends_on       = [azurerm_kubernetes_cluster.k8s]
+
+  set {
+    name  = "installCRDs"
+    value = true
+  }
+}
+
+resource "kubectl_manifest" "cert_manager_issuer" {
+  depends_on = [helm_release.cert_manager]
+
+  yaml_body = <<YAML
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod
+spec:
+  acme:
+    # You must replace this email address with your own.
+    # Let's Encrypt uses this to contact you about expiring
+    # certificates, and issues related to your account.
+    email: nclyde@skylight.digital
+    # ACME server URL for Let’s Encrypt’s staging environment.
+    # The staging environment won't issue trusted certificates but is
+    # used to ensure that the verification process is working properly
+    # before moving to production
+    server: https://acme-v02.api.letsencrypt.org/directory
+    privateKeySecretRef:
+      # Secret resource used to store the account's private key.
+      name: phdi-playground-issuer-account-key-prod
```

Comment:
Should this key be per-environment, instead of just `prod`?

Reply:
So this refers to the Let's Encrypt prod server (instead of staging for testing certs). But I think it would be fine to remove `prod` from the secret name, I'll go ahead and do that!

Reply:
Ahhh ok that makes much more sense, thanks!